mne/viz/ica.py

```diff
@@ -338,8 +345,18 @@ def plot_ica_properties(ica, inst, picks=None, axes=None, dB=True,
     else:
         kind = "Epochs"
 
+    
     epochs_src = ica.get_sources(inst)
-    ica_data = np.swapaxes(epochs_src.get_data()[:, picks, :], 0, 1)
+
+    # if reject is set, reject
+    data = epochs_src.get_data()
```

Comment:
this would work only for epochs? what about raw?

Reply:
these are epochs obtained from raw using _segment_raw function

Reply:
I'm not sure about this, but epochs_src is derived directly from the inst that can still be Raw (misleading variable name ?)

Reply:
Ok, did not understand this :)

Reply:
Yes, so this should work right ?

Reply:
yes